### ISSUE TITLE
Push more than once cause EUNSPECIFIED error on windows

### ISSUE THREAD

**Issue by jerson** [Microsoft/react-native-code-push#1175]
### Steps to Reproduce
1. in any windows application do push once
2. update the application normally
3. do a second push, when you try to synchronize, regardless of whether it is manual or automatically generates an unknown error

### Expected Behavior
What you expected to happen?
update the application normally

### Actual Behavior
What actually happens?
error returned

![screenshot from 2018-02-01 22-39-39](https://user-images.githubusercontent.com/325780/35715512-d65b2da0-07a0-11e8-94d8-3f5faf77a265.png)

### Environment

* react-native-code-push version: v5.2.1
* react-native version: v0.52.2
* Windows version: 10 aniversary update
* Does this reproduce on a debug build or release build? both
* Does this reproduce on a simulator, or only on a physical device? both

**Comment by svekl** [comment]
Hi @jerson,
I tried to reproduce your issue, but had no luck, I created new RN Windows project, created 5 CodePush updates and it works fine for me.  Looks like I'm missing some detail. May I ask you to create some small example project to reproduce this issue?

**Comment by iTOYS** [comment]
Closed as not reproduce.

If you still faced this issue, feel free to reopen issue and please provide info that i can use to reproduce

**Comment by leotm** [comment]
Getting similar, but on Mac emulator (simulator is fine).

This part errors

```tsx
update.install(codePush.InstallMode.ON_NEXT_RESUME)
```

Catching

<img width="1311" alt="Screenshot 2021-01-06 at 12 56 22" src="https://user-images.githubusercontent.com/1881059/103770955-c5a38100-501e-11eb-943f-8fca0d120c24.png">

**Comment by leotm** [comment]
Nvm fixed it, upgraded from 6.0.0, missed adding to `android/settings.gradle`

```gradle
include ':app', ':react-native-code-push'
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
```

**Comment by leotm** [comment]
On silent (non-mandatory) updates, continue to sometimes get the same `EUNSPECIFIED` error

<img width="287" alt="1" src="https://user-images.githubusercontent.com/1881059/103776917-0f449980-5028-11eb-8289-75f04fa151aa.png">

Each time, deleting the app, restarting metro bundler, re-building the app, disabling Fast Refresh, checking for error/success, if error repeating `ON_NEXT_RESUME` until it doesn't error then downloads from _codepush.appcenter.ms_

<img width="374" alt="2" src="https://user-images.githubusercontent.com/1881059/103776990-2a170e00-5028-11eb-84b3-528efd61e696.png">

Very tricky to debug, but mostly works first-go 👍

<img width="352" alt="Screenshot 2021-01-06 at 14 12 27" src="https://user-images.githubusercontent.com/1881059/103778071-a6f6b780-5029-11eb-9b3f-12b438abefc0.png">

Dealing with a 2yo React class, so will likely re-write it pronto if can.
